src/idiomatic/foundations-api-design/meaningful-doc-comments.md

```diff
@@ -0,0 +1,21 @@
+---
+minutes: 5
+---
+
+# Meaningful Doc Comments
+
+```rust,compile_fail
+/// API for the client // ❌ Lacks detail
+pub mod client {}
+
+/// Function from A to B // ❌ Redundant
+fn a_to_b(a: A) -> B {...}
+ 
+/// Connects to the database. // ❌ Lacks detail
+fn connect() -> Result<(), Error> {...}
+```
+
+Doc comments are the most common form of documentation developers engage with.
```

Comment:
Could this be moved to _before_ the code block to follow the style on most other pages? Compare with [Type Inference](https://google.github.io/comprehensive-rust/types-and-values/inference.html) for example to see the style I used for most of our slides:

> # Title
>
> One-line introduction
>
> ```rust
> // relevant code
> ```